src/sandbox/interpolation/atlas-parallel-interpolation.cc

```diff
@@ -245,7 +223,7 @@ int AtlasParallelInterpolation::execute(const AtlasTool::Args& args) {
             //            src_scalar_1( j ) = -std::tanh( y / 10. * std::cos( 50. / std::sqrt( x * x + y * y ) ) -
             //                                            x / 10. * std::sin( 50. / std::sqrt( x * x + y * y ) ) );
 
-            src_scalar_1(j) = vortex_rollup(lon, lat, 1.);
+            src_scalar_1(j) = util::vortex_rollup(lon, lat, 1., 1., false);
```

Comment:
Because of the lon, lat in radians, here we better pass `lonlat(j,0)` and `lonlat(j,1)` in degrees

Reply:
Done